collections/nginx-extended-modules.yml

```diff
@@ -0,0 +1,24 @@
+ident: nginx-extended-modules
+name: Nginx Extended Modules
+name_cn: Nginx扩展模块
+slug: /nginx-extended-modules
+items:
+  - https://github.com/NginxProxyManager/nginx-proxy-manager
+  - https://github.com/nginx-proxy/acme-companion
+  - https://github.com/evertramos/nginx-proxy-automation
+  - https://github.com/engintron/engintron
+  - https://github.com/vouch/vouch-proxy
+  - https://github.com/xavier-hernandez/goaccess-for-nginxproxymanager
+  - https://github.com/digitalocean/nginxconfig.io
+  - https://github.com/trimstray/nginx-admins-handbook
+  - https://github.com/andreapollastri/cipi
+  - https://github.com/schenkd/nginx-ui
+  - https://github.com/slomkowski/nginx-config-formatter
+  - https://github.com/caddyserver/nginx-adapter
+  - https://github.com/nginxinc/crossplane
+  - https://github.com/nginx-proxy/nginx-proxy
+  - https://github.com/allinurl/goaccess
+  - https://github.com/alibaba/tengine
+  - https://github.com/openresty/lua-nginx-module
+  - https://github.com/matsumotory/ngx_mruby
+  - https://gitee.com/winshining/nginx-http-flv-module
```

Comment:
Compare with https://github.com/oss-compass/compass-projects-information/commit/e1026d8af3a905980d29e17825420db53b7adf18:

```diff
+ ident: nginx-extended-modules
+ name: Nginx Extended Modules
+ name_cn: Nginx扩展模块
+ slug: /nginx-extended-modules
- ident: nginx-related
- name: Nginx Related
- name_cn: Nginx 相关
- slug: /nginx-related
items:
...
  - https://github.com/allinurl/goaccess
+  - https://github.com/alibaba/tengine
+  - https://github.com/openresty/lua-nginx-module
+  - https://github.com/matsumotory/ngx_mruby
+  - https://gitee.com/winshining/nginx-http-flv-module
```
The only difference is that the Collection Meta information has changed, and there are four more repositories.
In this case the Collection is highly overlapping. I understand that name changes or improvements are acceptable. But I don't understand why a new Collection should be created.

Reply:
Because we think that the first-level category of nginx should be server applications, not other categories, so we add 'Nginx extension module' under the server application category and inherit 'nginx-related' @normal-coder @eyehwan

Reply:
@normal-coder @eyehwan 
1. we are't sure what kind of problem will occur if we just remove an existed collection name; 
2. new name is better on reflecting the relationship between Nginx and those repos;  
Alternatively, it is easy to just adding some repos to "nginx-related.yml" if you thought it is better.